docs/core/index.rst

```diff
@@ -4,7 +4,6 @@ Core
 .. toctree::
     config
     auth
-    iterators
```

Comment:
I will, but in a separate PR (I plan to thoroughly document the entire google.api.core package)